include/openenclave/attestation/plugin.h

```diff
@@ -425,6 +427,64 @@ oe_result_t oe_verify_evidence(
  */
 oe_result_t oe_free_claims_list(oe_claim_t* claims, size_t claims_length);
 
+/**
+ * oe_get_registered_attester_format_ids
+ *
+ * Get the unique identifiers of all registered attesters.
+ *
+ * @param[out] format_ids The list of the format ids of the registered
+ * attesters.
+ * @param[out] format_ids_length The length of the format ids list.
+ * @retval OE_OK on success.
+ * @retval Otherwise, returns the error code the plugin's function.
+ */
+oe_result_t oe_get_registered_attester_format_ids(
```

Comment:
Is this an API for apps to call?  Why would an app ever want to call this?   New APIs should have sample code not just tests.